pkg/statistics/handle/autoanalyze/refresher/refresher.go

```diff
@@ -448,12 +448,11 @@ func CalculateChangePercentage(
 }
 
 func calculateTableSize(
-	tblInfo *model.TableInfo,
 	tblStats *statistics.Table,
 ) float64 {
 	tblCnt := float64(tblStats.RealtimeCount)
-	// TODO: Ignore unanalyzable columns.
-	colCnt := float64(len(tblInfo.Columns))
+	colCnt := float64(tblStats.ColNum())
```

Comment:
You should use the column count in `tableStats.ColAndIdxExistenceMap.colInfoMap`
The `tblStats.ColNum` only has the items that are loaded into the memory. It's possible that it's not full.

Reply:
Yes, I found out about it when I tried to add a new column. If we can get the full number from it, that would be great. Thank you!